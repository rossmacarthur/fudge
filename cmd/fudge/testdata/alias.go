package test

import test "github.com/rossmacarthur/fudge/errors"

var ErrTest = test.Sentinel("test error", "")
