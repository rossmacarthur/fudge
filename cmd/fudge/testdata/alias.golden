package test

import test "github.com/rossmacarthur/fudge/errors"

var ErrTest = test.Sentinel("test error", "ERR_52fdfc072182654f")
